game/script-farewells.rpy

```diff
@@ -547,6 +549,16 @@ label farewell_option_studying:
 
     return { "quit": None }
 
+label farewell_option_restart:
+    $ persistent.jn_player_restarted = True
+    n 1tnmsl "Going to restart the game,{w=0.2} [player]?"
+    n 4nlrss "Okay."
+    n 2nsgsg "Don't take too long!"
+    if Natsuki.isLove(higher=True):
+        n 2nwmsml "Love you!"
+
+    return { "quit": None }
+
```

Comment:
I think it'd go down better here if we take more advantage of the affinity levels, and lean into Natsuki's mischievous side more: while this dialogue works, it feels a bit toned down compared to how I'd feel Nat would respond.

I'd suggest having different variations based on a couple of levels:

```py
if Natsuki.isEnamored(higher=True):
    # Tease

    if Natsuki.isLove(higher=True):
        # Love you!
        
    else:
        # Later, player!

elif Natsuki.isAffectionate(higher=True):
    # Slight taunt

else:
    # Neutral/happy
```

No need to go overboard! Just a little more characterisation for a non-standard farewell that can give our players a smirk - don't be afraid to borrow inspiration from other farewells/dialogue!